### ISSUE TITLE
Atum Extensions: Extension promo images have no alt attribute

### ISSUE THREAD

**Issue by mbeganyi-a11y** [joomla/joomla-cms#48371]
### Steps to reproduce the issue

**NOTE**: Accessibility backlog issue and work-in-progress. This issue has been opened to account for all the issues flagged in the initial [Milestone 1a Audit Report](https://accessibility-project.joomla.org/results/Milestone-1a/milestone1/deliverable1/wcag-report). If this is a duplicate issue, it will be closed by the end of September 2026. 

Milestone 1a   finding ID | M1a-029
-- | --
Backlog   priority | High
Severity —   carried from Milestone 1a | Serious
Implementation   effort | High (API reliance)
Target   milestone(s) | MS2a, MS3a,   MS3b, MS7a, MS7b, MS8a, MS9a, MS10
WCAG 2.2   success criterion | 1.1.1   Non-text Content
Responsibility   tag(s) from report | Design,   Development
Affected   Joomla area/component | https://accessibility-project.joomla.org/results/Milestone-1a/milestone1/deliverable1/wcag-report#431d723d-f5b0-41df-80a7-0841c7eab8c4
Affected user   journey(s) | 07 – Installer Wizard (com_installer/Install)
Source   report/reference | Milestone 1a   WCAG audit report
STF reporting   reference | To be   assigned
Issue owner | To be   assigned
Status | Triage

<p class=MsoNormal>Audit finding<o:p></o:p></p>

<p class=MsoNormal>Extension promo images have no alt attribute<o:p></o:p></p>

<p class=MsoNormal>User impact<o:p></o:p></p>

<p class=MsoNormal>People using screen readers or other text alternatives may
miss the meaning, purpose, or target of important images or linked controls.<o:p></o:p></p>

<p class=MsoNormal>Remediation plan<o:p></o:p></p>

<p class=MsoNormal>Add the missing text alternative, accessible description, or
contextual relationship in markup so the information is exposed clearly to
assistive technologies.<o:p></o:p></p>

Steps to reproduce

1. Open the extension list.
2. Locate each promotional image.
3. Inspect alt attributes and accessible names.
4. Navigate the images or linked images with a screen reader.
5. Observe missing or unhelpful alternatives.

</body>

</html>

### Version

6.1

### Expected result

Promo images contain accurate alternative text.

### Actual result

No alternative text description provided.

### What device were you using? (eg, phone, laptop)

Desktop web

### What web browser were you using? (eg, Chrome, Safari)

Chrome, Firefox, Edge, Safari

### Assistive Technology?

VoiceOver, JAWS, NVDA

**Comment by brianteeman** [comment]
I might be wrong but I am fairly certain that this is the wrong repo for this as the code is created https://github.com/joomla/install-from-web-server and only consumed by joomla

**Comment by rdeutz** [comment]
@brianteeman is correct. The place to fix it is: https://github.com/joomla/install-from-web-server/blob/master/components/com_apps/layouts/apps/j4/extensions_imagegrid/grid.php.

But we agreed to create issues in the main repo representing the main application.

**Comment by brianteeman** [comment]
https://github.com/joomla/install-from-web-server/pull/85

**Comment by richard67** [comment]
Closing as not an issue with the Joomla CMS core, and a fix has been proposed at the right place. See previous comment with the link.

**Comment by brianteeman** [comment]
@richard67 @rdeutz make your mind up. Conflicting advice doesnt help @mbeganyi-a11y

**Comment by richard67** [comment]
> [@richard67](https://github.com/richard67) [@rdeutz](https://github.com/rdeutz) make your mind up. Conflicting advice doesnt help [@mbeganyi-a11y](https://github.com/mbeganyi-a11y)

Sorry, I haven't seen @rdeutz 's comment when closing.

**Comment by richard67** [comment]
PR https://github.com/joomla/install-from-web-server/pull/85 has been merged, and the changes have been deployed.

It might need a bit time until cache has been updated.

**Comment by mbeganyi-a11y** [comment]
> PR [joomla/install-from-web-server#85](https://github.com/joomla/install-from-web-server/pull/85) has been merged, and the changes have been deployed.
> 
> It might need a bit time until cache has been updated.

Thanks for the info on this, Richard; happy to hear that it has been merged! Makes sense to me that this issue can be closed.

**Comment by mbeganyi-a11y** [comment]
Closing issue as it has been resolved and merged.
